README.md

```diff
@@ -137,3 +137,180 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
 ```
 
 Here we have used a trick to convert the model from torch back to ONNX by defining the custom ``_ExpandExportToOnnx``.
+
+## Currently supported operations
+
```

Comment:
Move it to separate file operators.md and describe supported opsets (min - max opsets) in the header. Example of the  operators.md with good header can be found there https://github.com/onnx/onnx-tensorrt/blob/main/docs/operators.md

Reply:
Do not forget write note in readme.md about operators.md